cmd/dlv/tools.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+// List packages used by _scirpts
```

Comment:
s/_scirpts/scripts/

And is this because it's ignored now that it's been moved to `_scripts` and the underscore is blocking the Go tool from finding the deps?

Reply:
I don't know. Maybe it's just been broken for a while and we didn't notice.